### DISCUSSION TITLE
Project work suggestion

### DISCUSSION THREAD

**Discussion by SatyamJ-2003** [SartHak-0-Sach/Harkirat-Singh-course_code_and_notes#10] [Ideas]
Bhaiya you told us that you will soon be adding new projects in upcoming weeks in this repository which is also mentioned in the syllabus of the course.

My suggestion is that instead of putting the code here, since you have always emphasized on doing open source contributions, why don't you make a separate repository where you can add this code and since we are learning through your notes at same pace, we can contribute our ideas to it too helping us get our first open source contribution!!

What do you think?

**Comment by SartHak-0-Sach** [comment]
> Bhaiya you told us that you will soon be adding new projects in upcoming weeks in this repository which is also mentioned in the syllabus of the course.
> 
> My suggestion is that instead of putting the code here, since you have always emphasized on doing open source contributions, why don't you make a separate repository where you can add this code and since we are learning through your notes at same pace, we can contribute our ideas to it too helping us get our first open source contribution!!
> 
> What do you think?

Hi Satyam,
I think it is a great idea and I'll try to make all of these projects into open source repositories but what I want from you all is commitment towards up-skilling yourselves daily and then contribute if you have any ideas for improving existing design/features.

If you all can manage that then I'll be more than happy to host your first open source contribution repository😊
